packages/@pollyjs/adapter/src/index.js

```diff
@@ -205,7 +213,8 @@ export default class Adapter {
 
     this.assert(
       'Recording for the following request is not found and `recordIfMissing` is `false`.\n' +
-        `${pollyRequest.method} ${pollyRequest.url}\n`,
+        JSON.stringify({ identifiers, id, order }, null, 2) +
```

Comment:
We should add `url` as well since identifiers might not have all the details needed depending on the user's config (e.g. matchRequestsBy).